tests/basic.rs

```diff
@@ -273,3 +273,25 @@ async fn address_send_exercises_backpressure() {
         .now_or_never()
         .expect("be able to queue another message because the mailbox is empty again");
 }
+
+#[test]
+fn address_debug() {
+    let (addr1, _ctx): (Address<_>, Context<Greeter>) = Context::new(None);
```

Comment:
You could avoid the long type hint with turbo-fish syntax but that is a style preference :)


```suggestion
    let (addr1, _ctx) = Context::<Greeter>::new(None);
```